joomla/joomla-cms
# [4..0] Fixing Searchfilter Field in com_finder

Right now the Searchfilter field in the menu item does not work. Instead of that field, a simple text field is loaded. This fixes that. It also removes the Directories Field, which is used nowhere and is basically a copy of the Folderlist field that we have in core.

### How to test
1. Enable the smart search plugin, index all your content
2. Go to the search filters in the backend and create a new filter
3. Go to your Smart Search Menu item and see that the field "Search Filters" is just a text input.
4. Apply this patch
5. See that the Search Filters field now displays a list
